### DISCUSSION TITLE
Running kubeflow jobs

### DISCUSSION THREAD

**Discussion by m-mamdouhi** [kubernetes-sigs/kueue#2535] [Q&A] [answered]
I understand that batch jobs automatically generate a workload when created but what about kubeflow jobs? they are in the integrations and there seems to be support for them so how do I implement it? how can I get kueue to generate workloads for those jobs as well?

**Comment by tenzen-y** [comment, marked as answer]
Actually, the kubeflow jobs are natively supported in this upstream Kueue.
You can find the way how the kubeflow integrations work here: https://kueue.sigs.k8s.io/docs/tasks/run/kubeflow/

**Comment by tenzen-y** [comment]
As I can see, this issue, https://github.com/kubernetes-sigs/kueue/issues/2536, seems to have already been resolved.
